RecoMuon/GlobalTrackingTools/plugins/GlobalTrackQualityProducer.cc

```diff
@@ -311,5 +311,77 @@ GlobalTrackQualityProducer::trackProbability(Trajectory& track) const {
 
 }
 
+void GlobalTrackQualityProducer::fillDescriptions(edm::ConfigurationDescriptions& descriptions){
+  edm::ParameterSetDescription desc;
+  {
+    edm::ParameterSetDescription psd1;
+    psd1.setAllowAnything();
+    desc.add<edm::ParameterSetDescription>("ServiceParameters", psd1);
+  }
+  {
+    edm::ParameterSetDescription psd1;
+    psd1.setAllowAnything();
+    desc.add<edm::ParameterSetDescription>("GlobalMuonTrackMatcher", psd1);
+  }
+  desc.add<edm::InputTag>("InputCollection",edm::InputTag("globalMuons"));
+  desc.add<edm::InputTag>("InputLinksCollection",edm::InputTag("globalMuons"));
+  desc.add<std::string>("BaseLabel","GLB");
+  {
+    edm::ParameterSetDescription descGlbMuonRefitter;
+    descGlbMuonRefitter.add<edm::InputTag>("DTRecSegmentLabel" , edm::InputTag("dt1DRecHits"));
+    descGlbMuonRefitter.add<edm::InputTag>("CSCRecSegmentLabel" , edm::InputTag("csc2DRecHits"));
+    descGlbMuonRefitter.add<edm::InputTag>("GEMRecHitLabel" , edm::InputTag("gemRecHits"));
+    descGlbMuonRefitter.add<edm::InputTag>("ME0RecHitLabel" , edm::InputTag("me0Segments"));
+    descGlbMuonRefitter.add<edm::InputTag>("RPCRecSegmentLabel" , edm::InputTag("rpcRecHits"));
+
+    descGlbMuonRefitter.add<int>("MuonHitsOption", 1);
+    descGlbMuonRefitter.add<double>("PtCut", 1.0);
+    descGlbMuonRefitter.add<double>("Chi2ProbabilityCut", 30.0);
+    descGlbMuonRefitter.add<double>("Chi2CutCSC", 1.0);
+    descGlbMuonRefitter.add<double>("Chi2CutDT", 30.0);
+    descGlbMuonRefitter.add<double>("Chi2CutGEM", 1.0);
+    descGlbMuonRefitter.add<double>("Chi2CutME0", 1.0);
+    descGlbMuonRefitter.add<double>("Chi2CutRPC", 1.0);
+    descGlbMuonRefitter.add<int>("HitThreshold", 1);
```

Comment:
these changes to reimplement RecoMuon/GlobalTrackingTools/python/GlobalMuonRefitter_cff.py here is a dead end and probably does more harm than good.
I propose to do the same as for ```GlobalMuonTrackMatcher``` on L324 ("allowAnything" so that the corresponding PSet is picked up from a central space